integration/cli-tests/src/test/java/org/wso2/micro/integrator/cli/CliProxyserviceTestCase.java

```diff
@@ -39,7 +40,7 @@ public void miShowProxyAllTest() throws IOException {
         String artifactName_proxy_1[] = TestUtils.getArtifactList(outputForCLICommand).get(0).split(" ", 2);
         String artifactName_proxy_2[] = TestUtils.getArtifactList(outputForCLICommand).get(1).split(" ", 2);
 
-        Assert.assertEquals(artifactName_proxy_1[0], CLI_ADDRESS_PROXY);
+        Assert.assertEquals(artifactName_proxy_1[0], CLI_MEDIATOR_PROXY);
```

Comment:
This change was made to fix an already existing test failure and is unrelated to the purpose of this PR.